src/Dapr.Actors/Client/ActorProxyFactory.cs

```diff
@@ -6,23 +6,29 @@
 namespace Dapr.Actors.Client
 {
     using System;
+    using System.Text.Json;
     using Dapr.Actors.Builder;
     using Dapr.Actors.Communication.Client;
 
     /// <summary>
     /// Represents a factory class to create a proxy to the remote actor objects.
     /// </summary>
-    internal class ActorProxyFactory : IActorProxyFactory
+    public class ActorProxyFactory : IActorProxyFactory
     {
         private readonly IDaprInteractor daprInteractor;
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public readonly JsonSerializerOptions JsonSerializerOptions;
```

Comment:
This will always be null right? Nothing initializes it and it's readonly.

Reply:
You are correct - I just noticed this too. Once we decide on how we want to handle getting the `JsonSerializerOptions` in there I'll be sure to get unit tests created for any modifications.